### ISSUE TITLE
[Feature request] Graysale tray icon

### ISSUE THREAD

**Issue by FredFisher2** [brunofin/cohesion#35]
I would like a grayscale tray icon. Maybe toggleable since some people might not like it. OpenRGB has this option. The 2 tray icons I have (Cohesion being one of them) look so out of place. (Im on KDE, but this issue is likely applicable to gnome too)

**Comment by brunofin** [comment]
That makes sense and I'll look into it. Thanks for the report.

**Comment by oiimrosabel** [comment]
@brunofin Do you want me to make a grayscale version of the icon ? I still have the source Figma board (I can share that too if needed)

**Comment by brunofin** [comment]
Yes that would probably be a better option than the hack I put together for
it, that'd be much appreciated, thanks!
Regards,
Bruno Finger

**Comment by oiimrosabel** [comment]
Hey, @FredFisher2  !
I've realized a quick mockup of a grayscale Cohesion icon, what do you think ?

![Ref](https://github.com/user-attachments/assets/d31461a6-bd74-4a8f-b153-33e900846703)
![With KDE desktop](https://github.com/user-attachments/assets/3b4cbfd2-e275-4aec-99f0-17372bcb10da)

**Comment by ghost** [comment]
Sorry, I hate turning down good work, but I liked the current one, cause it fits with the rest of the tray icons, with just the black outline... But ultimately it's up to brunofin to decide. (Sorry, I feel terrible saying that)
